src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/DeleteCollectionCommand.java

```diff
@@ -17,11 +15,7 @@
 @Schema(description = "Command that deletes a collection if one exists.")
 @JsonTypeName(CommandName.Names.DELETE_COLLECTION)
 public record DeleteCollectionCommand(
-    @NotNull
-        @Size(min = 1, max = 48)
-        @Pattern(regexp = "[a-zA-Z][a-zA-Z0-9_]*")
-        @Schema(description = "Name of the collection")
-        String name)
+    @NotEmpty @Schema(description = "Required name of the Collection to remove") String name)
```

Comment:
Did we decide to lave `@NotEmpty` for "drop" commands and not "create" commands -- or why does this remain?

Reply:
Ok so this is needed since no other validation is performed. All good.